fmatch/matcher.py

```diff
@@ -73,6 +73,8 @@ def get_uuid_by_metadata(
         lookback_date: datetime = None,
         lookback_size: int = 10000,
         timestamp_field: str = "timestamp",
+        version_field: str = "ocpVersion",
```

Comment:
Seems like there is an attempt to change version field as well?

Reply:
yes, my version is also a different key so I needed to set that as well

Reply:
See the config I used in the bottom of the description in this PR: https://github.com/cloud-bulldozer/orion/pull/132#issue-3043405821

Reply:
Can we extract this string literal as a constant, please?